terraform/environment/dns.tf

```diff
@@ -0,0 +1,45 @@
+data "aws_route53_zone" "opg_metrics" {
+  provider = aws.management
+  name     = "metrics.opg.service.justice.gov.uk"
+}
+
+resource "aws_route53_record" "opg_metrics" {
```

Comment:
I find it helpful to leave a comment here to show what the record name should interpolate out to.
```suggestion
resource "aws_route53_record" "opg_metrics" {
# <environment.>api.metrics.opg.service.justice.gov.uk
```